source/gent/tier1_hortense.rst

```diff
@@ -768,28 +768,32 @@ This implies that the partitions ``cpu_rome``, ``cpu_rome_all``, ``cpu_rome_512`
 Depending on VSC plans, the ``a100_40`` partition may also disappear. However, there currently is no confirmation regarding this.
 
 
-Update to RHEL9 of Milan partitions
+Migration to RHEL9 operating system
 -----------------------------------
 
-To maintain operational safety, the operating system for the Milan CPU partition will be updated to a new major release.
-Red Hat Enterprise Linux version 9, RHEL9, (going up from 8) will be installed.
+To maintain operational safety, the operating system of Hortense will be updated to **Red Hat Enterprise Linux version 9 (RHEL9)**
+(going up from RHEL8).
 
-This implies that - end 2025, when the RHEL8 Rome partition is decommissioned - your software and/or workflow will need to be compliant with this OS version if you still want to run jobs.
-As of cutoff 2 in 2025, compatibility of your workflow/software with the new RHEL9 operating system will be a hard requirement.
+This implies that by November 2025, when the RHEL8 Rome partition is decommissioned, your software and/or workflow will need to be compliant with the RHEL9 OS if you still want to run jobs.
+As of the 2nd cutoff in 2025 (June'25), compatibility of your workflow/software with the new RHEL9 operating system will be a hard requirement.
 
 Please test your workflow and software as soon as possible and ensure that you are ready for this transition.
 
-To facilitate testing, we have made a small partition to run your tests: partition ``debug_milan_rhel9``
-This partition is SOLELY intended for testing your software/workflows.
-Do not run production jobs on this partition.
+
+Update to RHEL9 of Milan partitions
+***********************************
+
+To facilitate testing, we have made separate partitions of workernodes that run RHEL9 as operating systems:
+``debug_milan_rhel9`` and ``cpu_milan_rhel9``.
+These partitions are intended for testing your software/workflows on RHEL9.
```

Comment:
this is ambiguous. cpu_milan_rhel9 is not intended for testing your workflows. it is intended to run your jobs. :smile:  it is even the default partition.